Web/BLL/RockFood/Services/FoodService.cs

```diff
@@ -58,7 +58,7 @@ public List<IFoodable> Get()
 
         public IFoodable Get(int id)
         {
-            var food = _db.Foods.Find(id);
+            var food =  _db.Foods.Find(id);
```

Comment:
var food =  _db.Foods.Find(id);
->
var food = _db.Foods.Find(id);

one space is eniuth

Reply:
is enough